.vscode/launch.json

```
@@ -0,0 +1,66 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch: Electron",
+      "type": "node",
+      "request": "launch",
+      "protocol": "inspector",
+      "sourceMaps": true,
+      "presentation": {
+        "hidden": false,
+        "group": "Insomnia",
+        "order": 1
+      },
+      "cwd": "${workspaceFolder}/packages/insomnia-app",
+      "runtimeExecutable": "${workspaceFolder}/packages/insomnia-app/node_modules/.bin/electron",
+      "runtimeArgs": ["--remote-debugging-port=9222", "."],
+      "outputCapture": "std",
+      "sourceMapPathOverrides": {
+        "webpack:///./*": "${workspaceFolder}/packages/insomnia-app/app/*"
+      },
+      "windows": {
+        "type": "node",
+        "request": "launch",
+        "name": "Launch: Electron",
+        "runtimeExecutable": "${workspaceFolder}/packages/insomnia-app/node_modules/.bin/electron.cmd"
+      },
+      "env": {
+        "TS_NODE_PROJECT": "tsconfig.webpack.json",
+        "NODE_ENV": "development",
+        "ELECTRON_IS_DEV": "1"
+      }
+    },
+    {
+      "name": "Attach: Electron Renderer",
+      "type": "chrome",
```

It looks like the chrome extension has been deprecated in favor of https://github.com/microsoft/vscode-js-debug which is now built in. This supports chrome, supposedly, but I couldn't get it to work 🤔 seems like setting this type to `pwa-chrome` worked, and that's one of the supported types registered by this built-in extension (https://github.com/microsoft/vscode-js-debug/blob/main/OPTIONS.md). Maybe the docs are out of date? How did you get this to work with `"type": "chrome"`?

```suggestion
      "type": "pwa-chrome",
```